packages/repo/src/mst/mst.ts

```diff
@@ -180,16 +180,16 @@ export class MST implements DataStore {
   // -------------------
 
   // Persist the MST to the blockstore
-  async save(): Promise<CID> {
+  async stage(): Promise<CID> {
```

Comment:
What's the right way to think about this name change in the MST?  Is the comment above "persist the MST..." accurate?

Is the idea is that the MST itself doesn't really have the ability to commit, since that will happen in coordination with the repo?

Reply:
Yup exactly. There are actually a couple stages to the block saving process here:
- node in MST is edited: we don't hash anything, just maintain in-memory references (this actually gives us ~100x speedup)
- MST is `staged`: we're reasonably confident this is the final state of the MST after some set of mutations. Traverse it & stage any changed nodes to the blockstore
- blockstore is saved: a valid commit was produced so the MST nodes are now canonical & we can save them to the permanent store

Reply:
Cool cool that all tracks, thanks!